doc/admin.rst

```diff
@@ -0,0 +1,198 @@
+
+(This port is *not* where the ArchiveBot Twitter bot gets its data; that's a different daemon.)
+
+Logs from this pipeline manager are stored in ``plumbing/log-firehose``.  Someday this log firehose could be replaced with Redis pubsub.
+
+
+tmux pane 2: pipeline log analyzer and log trimmer
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This pane manages the pipeline log analyzer and the pipeline log trimmer.
+
+The log analyzer looks at updates coming off the firehose and classifies them as HTTP 1xx, 2xx, etc, or network error.
+
+The log trimmer is an artifact of how ArchiveBot stores logs, could probably be removed someday.  It gets rid of old logs from Redis to prevent out-of-memory errors.
+
+
+tmux pane 3: web-based dashboard
+++++++++++++++++++++++++++++++++
+
+This pane runs the web-based ArchiveBot dashboard, which is publicly viewable at:
+http://dashboard.at.ninjawedding.org/
+
+This tmux pane is split into two parts on the screen, top and bottom.  The top pane shows the throughput of the dashboard web socket, which is the rate of data flowing from the log firehose to the dashboard.
+
+The web-based dashboard has a small unknown memory leak, so the bottom pane runs and monitors ivan's “dashboard killer” daemon. It constantly polls the dashboard to see if it's alive, and it prints a dot if it was a success (dashboard was alive and responded).  If the dashboard does not respond, probably because of that small memory leak, then this daemon kills it and automatically re-spawns it.
+
+
+tmux pane 4: IRC bot
+++++++++++++++++++++
+
+This pane runs the actual ArchiveBot, which is an IRC bot that sits in the channel #archivebot on EFNet and listens for Archive Team volunteers feeding it commands about what websites to archive.
+
+Usually, there's not much that an administrator will need to do for this. If the bot gets kicked off EFnet, it will try to reconnect on its own. However, EFnet sometimes has the tendency to netsplit (disconnect from some IRC noces in a disorganized manner). If that happens, the bot might try to rejoin a server that's been split, in which case the bot might need to be "kicked" (restarted and reconnected to the IRC server).
+
+If you need to kick it, hit ``^C`` in this pane to kill the non-responding bot. Then hit the ``Up arrow key`` to show the last command that had been typed into bash, which is usually the one that invokes the bot. You can then adjust that command if you need to (such as possibly changing the server), and then hit enter to re-run that command and reconnect the bot to EFnet.
+
+
+tmux pane 5: redis-cli console
+++++++++++++++++++++++++++++++
+
+This is the console for running redis-cli commands.  It might get closed down, because it's rarely used.
```

Comment:
It might be okay to just omit this section -- panes like this pop in and out all the time.